deployment/local-dev/pgadmin-deployment.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: pgadmin-auth
+  namespace: languagedepot
+stringData:
+  PGADMIN_DEFAULT_EMAIL: 'admin@test.com'
+  PGADMIN_DEFAULT_PASSWORD: 'pass'
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: pgadmin-config
+data:
+  servers.json: |
+    {
+      "Servers": {
+        "1": {
+          "Name": "lexbox",
+          "Group": "Servers",
+          "Port": 5432,
+          "Username": "postgres",
+          "PassFile": "/lexbox",
+          "Host": "db",
+          "SSLMode": "prefer",
+          "MaintenanceDB": "postgres"
+        }
+      }
+    }
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: pgadmin
+  namespace: languagedepot
+  labels:
+    app: pgadmin
+spec:
+  type: ClusterIP
+  clusterIP: None
+  selector:
+    app: pgadmin
+  ports:
+    - name: http
+      protocol: TCP
+      port: 80
+
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: pgadmin
+  namespace: languagedepot
+  labels:
+    app: pgadmin
+spec:
+  selector:
+    matchLabels:
+      app: pgadmin
+  strategy:
+      type: RollingUpdate
```

Comment:
RollingUpdate is the default, so if I leave this out then it will act the same way. But yeah, switching to Recreate is probably fine; downtime on the pgadmin container really doesn't matter.

Reply:
Done in commit 47585fdd.